Lib/random.py

```diff
@@ -849,7 +849,7 @@ class SystemRandom(Random):
     """
 
     def random(self):
-        """Get the next random number in the range [0.0, 1.0)."""
+        """Return the next random floating point number in the range ``0.0 <= X < 1.0``"""
```

Comment:
No backticks here.  This is a docstring and doesn't use markup.

Reply:
Also, let's keep the wording as-is and only change the  interval.  I know it's a little different that the mains but that is okay.